src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj

```diff
@@ -34,7 +34,7 @@
   </ItemDefinitionGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.8.2" />
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="2.8.2" />
```

Comment:
Don't code fixes require a reference to Workspaces?

Reply:
Microsoft.CodeAnalysis.Common is the root compiler assembly. Microsoft.CodeAnalysis is a higher-level package that includes both languages' compiler packages and workspaces packages.